statcommander_test.go

```diff
@@ -59,6 +60,9 @@ func TestHelperProcessSuccess(t *testing.T) {
 		return
 	}
 
+	// Drain input if there is any.
+	_, _ = ioutil.ReadAll(os.Stdin)
```

Comment:
I'm unsure whether or not you need them, but I agree that stating explicitly that you don't care about the return data is a good thing here.